config/runtime/server.go

```diff
@@ -338,7 +352,7 @@ func NewServerConfiguration(conf *config.Couper, log *logrus.Entry, memStore *ca
 				return nil, err
 			}
 
-			epHandler = middleware.NewCORSHandler(corsOptions, epHandler)
+			epHandler = middleware.NewCORSHandler(corsOptions, allowedMethodsHandler.MethodAllowed, epHandler)
```

Comment:
Move the MethodAllowed func argument to `corsOptions` then we do not have to call explicitly with `nil` for the other cases.

Reply:
The `allowedMethodsHandler.MethodAllowed` (or `nil`) is now passed to `NewCORSOptions()` instead.

Reply:
I see. At least we have all options at one place.